`experience` objective event and condition where reworked (#2179)
BetonQuest/BetonQuest: docs/Documentation/Objectives-List.md

## Experience: `experience`

> This objective can by completed by reaching specified amount of experience levels.

Reviewer: ```suggestion
This objective can be completed by reaching the specified amount of experience levels.
```

You can also define decimal numbers, for example `experience 1.5` will complete when the player reaches 1.5 experience levels or more.

> If you want to check for an absolute amount of experience points by converting it to levels.

Reviewer: What is this sentence supposed to mean?

> The objective is checked every time the player gets experience by a natural way, such as killing mobs or mining blocks
> or if the player reaches a new level, also by a command or other plugin.

Reviewer: ```suggestion
The objective is checked every time the player gets experience naturally, such as killing mobs or mining blocks.
Additionally, it is checked if the player reaches a new level in any way (vanilla level up, commands or other plugins).
```

> You can use the `notify` keyword to display a message each

Reviewer: Please document the behaviour of the objective when the player already has the required amount of XP when the objective is started.
